packages/opentelemetry-node/lib/sdk.js

```diff
@@ -144,37 +144,19 @@ function startNodeSDK(cfg = {}) {
     }
 
     // TODO: How does this differ from `OTEL_METRICS_EXPORTER=none` in sdk-node?
-    const metricsDisabled =
-        getBooleanFromEnv('ELASTIC_OTEL_METRICS_DISABLED') ?? false;
-    if (!metricsDisabled) {
-        const metricsExportProtocol =
-            getStringFromEnv('OTEL_EXPORTER_OTLP_METRICS_PROTOCOL') ||
-            getStringFromEnv('OTEL_EXPORTER_OTLP_PROTOCOL') ||
-            'http/protobuf';
-        let metricExporterType =
-            exporterPkgNameFromEnvVar[metricsExportProtocol];
-        if (!metricExporterType) {
-            log.warn(
-                `Metrics exporter protocol "${metricsExportProtocol}" unknown. Using default "http/protobuf" protocol`
-            );
-            metricExporterType = 'proto';
-        }
-        log.trace(`Metrics exporter protocol set to ${metricsExportProtocol}`);
-        const {OTLPMetricExporter} = require(
-            `@opentelemetry/exporter-metrics-otlp-${metricExporterType}`
-        );
-
-        // https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#periodic-exporting-metricreader
-        const metricsInterval =
-            getNumberFromEnv('OTEL_METRIC_EXPORT_INTERVAL') ?? 60000;
-        const metricsTimeout =
-            getNumberFromEnv('OTEL_METRIC_EXPORT_TIMEOUT') ?? 30000;
-
-        defaultConfig.metricReader = new metrics.PeriodicExportingMetricReader({
-            exporter: new OTLPMetricExporter(),
-            exportIntervalMillis: metricsInterval,
-            exportTimeoutMillis: metricsTimeout,
-        });
+    // The implementation in SDK does treats the undefined and 'none' value
+    // as same. https://github.com/open-telemetry/opentelemetry-js/blob/dac72912b3a895c91ee95cfa39a22a916411ba4c/experimental/packages/opentelemetry-sdk-node/src/sdk.ts#L117
+    // According to https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#exporter-selection
+    // the default should be 'otlp' so IMHO the implementation is wrong
+
+    // We need to set the default value if env var is not defined until it's fixed in
+    // upstream SDK
```

Comment:
You are right! Good catch. That was a little surprising.
nit: It would be nice to open an issue (or PR) on sdk-node and refer to it here.